trends_data/src/commonMain/kotlin/net/thechance/mena/trends/data/repository/ReelsRepositoryImpl.kt

```diff
@@ -54,6 +54,14 @@ internal class ReelsRepositoryImpl(
         }.results?.map { it.toEntity() } ?: emptyList()
```

Comment:
u can use .orEmpty() instead
```suggestion
   }.results?.map { it.toEntity() } .orEmpty()